src/main/java/uk/gov/ons/census/fwmt/rmadapter/service/impl/RMAdapterServiceImpl.java

```diff
@@ -41,5 +45,15 @@ public void sendJobRequest(ActionInstruction actionInstruction) throws GatewayEx
             .triggerEvent(actionInstruction.getActionCancel().getCaseId(), CANONICAL_CANCEL_FAILED, LocalTime.now());
       }
     }
+    else if (actionInstruction.getActionUpdate() != null){
+      if (!StringUtils.isEmpty(actionInstruction.getActionUpdate().getPause().getId())) {
+        jobServiceProducer.sendMessage(CanonicalJobHelper.newUpdateJob(actionInstruction));
+        gatewayEventManager
+            .triggerEvent(actionInstruction.getActionUpdate().getPause().getId(), CANONICAL_UPDATE_SENT, LocalTime.now());
+      } else {
+        gatewayEventManager
+            .triggerEvent("00000000-0000-0000-0000-000000000000", CANONICAL_UPDATE_FAILED, LocalTime.now());
```

Comment:
what are you doing here?

Reply:
This has been updated as discussed earlier